package.json

```diff
@@ -248,6 +258,16 @@
           "command": "quarkusTools.deployToOpenShift.short",
           "when": "quarkusProjectExistsOrLightWeight && resourceExtname == .java",
           "group": "navigation@10"
+        },
+        {
+          "command": "qute.validation.enabled.toggle.on",
+          "when": "quarkusProjectExistsOrLightWeight && editorLangId == qute-html && quteValidationEnabled",
+          "group": "navigation@9"
+        },
+        {
+          "command": "qute.validation.enabled.toggle.off",
+          "when": "quarkusProjectExistsOrLightWeight && editorLangId == qute-html && !quteValidationEnabled",
```

Comment:
need to check editorLangId against all 4 of the qute flavors (html, txt, yaml, json)